smoke-tests/matrix/build.gradle

```diff
@@ -80,8 +80,7 @@ def windowsTargets = [
     [version: ["7.0.107"], vm: ["hotspot", "openj9"], jdk: ["8"], args: [majorVersion: "7"]],
     [version: ["8.5.60"], vm: ["hotspot", "openj9"], jdk: ["8", "11"], args: [majorVersion: "8"]],
     [version: ["9.0.40"], vm: ["hotspot", "openj9"], jdk: ["8", "11"], args: [majorVersion: "9"]],
-    [version: ["10.0.4"], vm: ["hotspot"], jdk: ["11", "16", "17"], args: [majorVersion: "10"], war: "servlet-5.0"],
-    [version: ["10.0.4"], vm: ["openj9"], jdk: ["11", "16"], args: [majorVersion: "10"], war: "servlet-5.0"]
+    [version: ["10.0.4"], vm: ["hotspot", "openj9"], jdk: ["11", "15"], args: [majorVersion: "10"], war: "servlet-5.0"],
```

Comment:
Is it correct that we downgrade 16/17 to 15 here?

Reply:
The `TomcatSmokeTest` class doesn't run 16/17 tests anyway: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/smoke-tests/src/test/groovy/io/opentelemetry/smoketest/TomcatSmokeTest.groovy

(We should upgrade to 17 though; preferably both for linux and windows)

Reply:
> We should upgrade to 17 though; preferably both for linux and windows

👍 I'll do this in follow-up PR, this PR just gets us back to steady state.